packages/ckeditor5-core/src/editor/editor.js

```diff
@@ -220,6 +220,31 @@ export default class Editor {
 		 */
 		this.keystrokes = new EditingKeystrokeHandler( this );
 		this.keystrokes.listenTo( this.editing.view.document );
+
+		/**
+		 * Current number of editors instances created.
+		 *
+		 * @private
+		 * @type {Number}
+		 */
+		Editor._editorCount = Editor._editorCount ? ++Editor._editorCount : 1;
```

Comment:
I'd solve this file like this

```javascript
diff --git a/packages/ckeditor5-core/src/editor/editor.js b/packages/ckeditor5-core/src/editor/editor.js
index 5f1d06d387..3988704360 100644
--- a/packages/ckeditor5-core/src/editor/editor.js
+++ b/packages/ckeditor5-core/src/editor/editor.js
@@ -221,30 +221,13 @@ export default class Editor {
         this.keystrokes = new EditingKeystrokeHandler( this );
         this.keystrokes.listenTo( this.editing.view.document );

-        /**
-         * Current number of editors instances created.
-         *
-         * @private
-         * @type {Number}
-         */
-        Editor._editorCount = Editor._editorCount ? ++Editor._editorCount : 1;
-
         /**
          * A unique editor's name.
          *
          * @observable
          * @member {String} #name
          */
-        this.name = this.generateEditorLabel();
-    }
-
-    /**
-     *  Static getter for the _editorCount property.
-     *
-     * @returns {module:core/editor/editor~Editor._editorCount}
-     */
-    static get editorCount() {
-        return Editor._editorCount;
+        this.name = this._generateEditorLabel();
     }

     /**
@@ -464,15 +447,22 @@ export default class Editor {
      * It will be based on the editor.config.get( 'label' ), otherwise a name
      * pattern of `'editor{n}'` will be used.
      *
-     * @readonly
+     * @private
      * @returns {String} label
      */
-    generateEditorLabel() {
-        const label = this.config.get( 'label' ) ? this.config.get( 'label' ) : 'editor' + Editor.editorCount;
+    _generateEditorLabel() {
+        const label = this.config.get( 'label' ) ? this.config.get( 'label' ) : 'editor' + ++Editor._editorCount;

         return label;
     }

+    /**
+     * Editor instance counter for labeling and accessibility.
+     *
+     * @private
+     * @property {number} #_editorCount
+     */
+
     /**
      * Creates and initializes a new editor instance.
      *
@@ -490,6 +480,8 @@ export default class Editor {
      */
 }

+Editor._editorCount = 0;
+
 mix( Editor, ObservableMixin );

 /**
```